pkgs/top-level/all-packages.nix

```diff
@@ -16695,6 +16695,10 @@ with pkgs;
 
   slack = callPackage ../applications/networking/instant-messengers/slack { };
 
+  wrapSlackCli = callPackage ../tools/networking/slack-cli/wrapper.nix { };
+  slack-cli-unwrapped = callPackage ../tools/networking/slack-cli { };
+  slack-cli = wrapSlackCli slack-cli-unwrapped;
+
```

Comment:
Is there a reason to add all three of these to `all-packages.nix`?

Reply:
A reason one might prefer a wrapper derivation is when the unwrapped one takes a long time to rebuild but the wrapper should have a quick turnaround time. Changing something in the wrapper doesn't affect the unwrapped one at all and can therefore be very quick.

Reply:
https://github.com/NixOS/nixpkgs/pull/45055#discussion_r210268395

Reply:
@Infinisil what do you think of https://github.com/NixOS/nixpkgs/pull/45055#discussion_r210268395?